### DISCUSSION TITLE
Add keybind to apply auto-offset

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#30699] [Ideas]
As there has been some complaints about not being able to apply auto-offset after the recent change on quick retry, I think this might be a reasonable stopgap for those who don't want to use normal retry.

An alternative could be to add a keybind to normal retry instead, but that sounds somewhat hacky for me UX-wise. That would allow people to retain their muscle memory tho, so might be something to consider.

**Comment by peppy** [comment]
Adding a key bind to apply something you can't see isn't something I can agree with.

A better solution is making settings available during quick retry based on user interaction. Will be considered in the future.

**Reply by felipemarins** [reply]
Yeah, I didn't consider this to be an issue when I was writing the discussion, but I agree that it would be weird to apply the auto-offset without the user being able to tell visually if it is even necessary.

I guess making the offset settings appear during quick retry may be the preferable solution, like you said (maybe by holding Ctrl, like it works for making the HUD visible when you use Shift-Tab).
